src/scene/export.ts

```diff
@@ -39,29 +43,59 @@ export function exportToCanvas(
 
   const tempCanvas: any = createCanvas(width, height);
 
+  const sceneState = {
+    viewBackgroundColor: exportBackground ? viewBackgroundColor : null,
+    scrollX: normalizeScroll(-minX + exportPadding),
+    scrollY: normalizeScroll(-minY + exportPadding),
+    zoom: 1,
+    remotePointerViewportCoords: {},
+    remoteSelectedElementIds: {},
+    shouldCacheIgnoreZoom: false,
+    remotePointerUsernames: {},
+  };
+
   renderScene(
     elements,
     appState,
     null,
     scale,
     rough.canvas(tempCanvas),
     tempCanvas,
-    {
-      viewBackgroundColor: exportBackground ? viewBackgroundColor : null,
-      scrollX: normalizeScroll(-minX + exportPadding),
-      scrollY: normalizeScroll(-minY + exportPadding),
-      zoom: 1,
-      remotePointerViewportCoords: {},
-      remoteSelectedElementIds: {},
-      shouldCacheIgnoreZoom: false,
-      remotePointerUsernames: {},
-    },
+    sceneState,
     {
       renderScrollbars: false,
       renderSelection: false,
       renderOptimizations: false,
     },
   );
+
+  if (addWatermark) {
+    const context = tempCanvas.getContext("2d")!;
+    const text = "Made in Excalidraw";
+    const font = "20px Virgil";
+    const { width: textWidth, height: textHeight } = measureText(text, font);
+
+    renderElement(
+      newTextElement({
+        text,
+        font,
+        textAlign: "center",
+        x: maxX - textWidth / 2,
+        y: maxY - textHeight / 2,
+        strokeColor: "#000000",
+        backgroundColor: "transparent",
+        fillStyle: "hachure",
+        strokeWidth: 1,
+        roughness: 1,
+        opacity: 60,
```

Comment:
can we remove the default values?

Reply:
Currently these fields are required for `ElementConstructorOpts`. I'm not sure where the default values could come from?